_migration-assistant/migration-phases/live-traffic-migration/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Live traffic migration
+nav_order: 99
+parent: Migration phases
```

Comment:
Set `has_toc` to false so related articles don't show. Otherwise, the steps are listed twice.